test/Scope/Positive.hs

```diff
@@ -50,7 +50,7 @@ testDescr PosTest {..} = helper renderCodeNew
                 entryPoint <- testDefaultEntryPointIO tRoot file'
 
                 let evalHelper :: Text -> Sem (PipelineEff PipelineAppEffects) a -> IO (PipelineResult a)
-                    evalHelper input m = snd <$> testRunIO entryPoint {_entryPointStdin = Just input} m
+                    evalHelper input' m = snd <$> testRunIO entryPoint {_entryPointStdin = Just input'} m
```

Comment:
name change

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582